dpe/src/dpe_instance.rs

```diff
@@ -41,19 +43,40 @@ pub struct DpeEnv<'a, T: DpeTypes + 'a> {
     pub platform: T::Platform<'a>,
 }
 
+#[repr(C)]
+#[derive(
+    Debug,
+    PartialEq,
+    Eq,
+    zerocopy::FromBytes,
+    zerocopy::IntoBytes,
+    zerocopy::Immutable,
+    zerocopy::KnownLayout,
+    Zeroize,
+)]
+pub struct DpeInstanceFlags(pub u32);
+
+bitflags! {
+    impl DpeInstanceFlags: u32 {
+        /// Can only successfully execute the initialize context command for non-simulation (i.e.
+        /// `InitializeContext(simulation=false)`) once per reset cycle.
+        ///
+        /// NOTE: This flag is for internal use only.
```

Comment:
Since flags are passed in the constructor, I think we may want to avoid tracking initialization state in flags. Probably makes sense as a separate internal bool.

Reply:
If you want to avoid increasing DpeInstance size, these flags can probably be a u16

Reply:
Done, moving it out of the flags uses 70 more bytes of code size 😭